src/verify_cert.rs

```diff
@@ -158,20 +162,39 @@ fn check_signatures(
     supported_sig_algs: &[&SignatureAlgorithm],
     cert_chain: &Cert,
     trust_anchor: &TrustAnchor,
+    revocation: &Option<RevocationCheckOptions>,
 ) -> Result<(), Error> {
     let mut spki_value = untrusted::Input::from(trust_anchor.spki);
+    let mut issuer_subject = untrusted::Input::from(trust_anchor.subject);
     let mut cert = cert_chain;
     loop {
         signed_data::verify_signed_data(supported_sig_algs, spki_value, &cert.signed_data)?;
 
-        // TODO: check revocation
-
         match &cert.ee_or_ca {
             EndEntityOrCa::Ca(child_cert) => {
+                if let Some(revocation_opts) = revocation {
```

Comment:
Can we deduplicate the `check_crl()` calls here? They seem to be identical?

Reply:
Good catch, can do.

Reply:
Fixed.